apps/src/componentLibrary/tabs/Tabs.tsx

```diff
@@ -73,6 +73,11 @@ const Tabs: React.FunctionComponent<TabsProps> = ({
     [setSelectedValue, onChange]
   );
 
+  // Reset selected tab whenever default selected value changes.
+  useEffect(() => {
+    setSelectedValue(defaultSelectedTabValue);
+  }, [defaultSelectedTabValue]);
+
```

Comment:
FYI @levadadenys added logic here to reset the selected tab whenever the default is updated.